### PR TITLE
[stable10] Improving token_auth_enforced description text

### PR DESCRIPTION
**Pull request by mmattel**
## Description

Backport of https://github.com/owncloud/core/pull/31101

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by phil-davis** [comment]
Local rebase and force push done to trigger CI

**Comment by mmattel** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

File: config/config.sample.php [hunk 1]
```diff
@@ -216,11 +216,11 @@
...
- * Enforce token authentication for clients, which blocks requests using the user
- * password for enhanced security. Users need to generate tokens in personal settings
- * which can be used as passwords on their clients.
- */
-'token_auth_enforced' => false,
+ * Enforces token only authentication for apps and clients connecting to ownCloud. 
+ * If enabled, all access requests using the users password are blocked for enhanced security. 
+ * Users have to generate special app-passwords (tokens) for their apps or clients in their personal 
+ * settings which are further used for app or client authentication. Browser logon is not affected. 
+ */'token_auth_enforced' => false,
```
Comment (phil-davis) [inline config/config.sample.php]: A new line went missing here.

Comment (mmattel) [reply config/config.sample.php]: (zensored) :see_no_evil: thanks

**Comment by mmattel** [comment]
@phil-davis fixed

File: config/config.sample.php [hunk 2]
```diff
@@ -216,11 +216,12 @@
...
- * Enforce token authentication for clients, which blocks requests using the user
- * password for enhanced security. Users need to generate tokens in personal settings
- * which can be used as passwords on their clients.
+ * Enforces token only authentication for apps and clients connecting to ownCloud. 
+ * If enabled, all access requests using the users password are blocked for enhanced security. 
+ * Users have to generate special app-passwords (tokens) for their apps or clients in their personal 
+ * settings which are further used for app or client authentication. Browser logon is not affected. 
  */
-'token_auth_enforced' => false,
+ 'token_auth_enforced' => false,
```
Comment (phil-davis) [inline config/config.sample.php]: It still shows a 1-space diff here.

It should be possible to just cherry-pick the commit from master back to stable10, and not need to do any editing.

Comment (mmattel) [reply config/config.sample.php]: fix is done. It´s embarrassed... thanks again

**Review by phil-davis — APPROVED** [review]

**Pull request merged**